tests/modules/asset_groups/resources/test_permissions.py

```diff
@@ -122,10 +117,7 @@ def test_create_patch_asset_group(
 
     finally:
         tus_utils.cleanup_tus_dir(transaction_id)
-        try:
-            delete_remote(str(temp_asset_group.guid))
-        except GitlabInitializationError:
-            pass
+        delete_remote(str(temp_asset_group.guid))
```

Comment:
same comment for all of the delete_remotes in the finally blocks

Reply:
I removed `delete_remote` from clean up code